### DISCUSSION TITLE
Dropdown box for changing graph scaling

### DISCUSSION THREAD

**Discussion by MrDragon420** [jneilliii/OctoPrint-BedLevelVisualizer#396] [Ideas]
I cant amek sense of how to make the colors different, a drop down with preselected boxes would be awesome

**Comment by jneilliii** [comment]
That's a nice idea for the standard colorscales, but not so much for the percentage based scaling (default settings for example).  The page here describes those options, and the first custom colorscale listed is what the plugin is doing by default.

https://plotly.com/javascript/colorscales/
